tests/nn/moe/test_moe_layer.py

```diff
@@ -73,6 +73,56 @@ def test_forward(device):
     assert torch.allclose(input, output)
 
 
+# Test Gate which round-robin routes tokens to experts
+class RoundRobinGate(torch.nn.Module):
+    def __init__(self, model_dim, num_experts):
+        super().__init__()
+        self.model_dim = model_dim
+        self.num_experts = num_experts
+
+    def forward(self, input):
+        g, s, m = input.shape
+        capacity = 2 * s // self.num_experts
```

Comment:
it seems that "//" should be "/" because if it is not divided evenly, then you should get an error anyway. "//" will mask error?

Reply:
Added an assertion.